### PR TITLE
The puppeteer image with node 14

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- adds Dockerfile/Makefile/Readme to create the puppeteer image that is used in the gitlab runners

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: tests/e2e/docker/Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	)
+
+shell: ## runs bash in the image to allow manual testing
+	@docker run -it $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) bash
+
+push: ## publish the image
+	# pushing $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)...
+	@docker push $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)
+
+guard-%:
```
Comment (pcrespov) [inline tests/e2e/docker/Makefile:25]: NICE!

File: tests/e2e/docker/Makefile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	@if [ "${${*}}" = "" ]; then echo "Environment variable $* not set"; exit 1; fi
+
+test: guard-URL guard-USER guard-PASSWORD ## test e2e sleepers scenario by clone osparc-simcore and running the sleepers tutorial (needs URL, USER and PASSWORD)
+	@docker run -it $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) "git clone https://github.com/ITISFoundation/osparc-simcore.git; npm i osparc-simcore/tests/e2e/; node osparc-simcore/tests/e2e/tutorials/sleepers.js $(URL) $(USER) $(PASSWORD)"
```
Comment (pcrespov) [inline tests/e2e/docker/Makefile:29]: why not mount the repo dir instead of ``git clone``??

**Review by pcrespov — APPROVED** [review]
just minors. Very nice!

File: tests/e2e/docker/Makefile (hunk 2 above)
Comment (pcrespov) [reply tests/e2e/docker/Makefile:29]: I understand that this way of using puppeteer is only for headless test and therefore will not allow the "demo" mode in which you see how the browser opens and the mouse moves over the UI, right?

Comment (sanderegg) [reply tests/e2e/docker/Makefile:29]: you are actually right. I basically copy-pasted the e2e recipe. will improve that.

**Review by odeimaiz — APPROVED** [review]

**Pull request merged**